coala_langserver/langserver.py

```diff
@@ -102,7 +93,46 @@ def send_diagnostics(self, path, diagnostics):
             'uri': 'file://{0}'.format(path),
             'diagnostics': _diagnostics,
         }
-        self.send_notification('textDocument/publishDiagnostics', params)
+        self._endpoint.notify('textDocument/publishDiagnostics', params=params)
+
+
+def start_tcp_lang_server(bind_addr, port, handler_class):
```

Comment:
unless handler_class is optional, it should be the first arg like `start_io_lang_server`, for consistency sake.